### PR TITLE
Transaction row details and layout improvements

### PR DESCRIPTION
**Pull request by mvaivre**

### CODE REVIEW

**Review by nop33 — COMMENTED** [review]
ok, respect, I love this

image

Have you thought of using the ellipsis component that we have in the desktop wallet, where the `...` is added in the middle? Have a look at [`HashEllipsed`. (in branch `token-list`)](https://github.com/alephium/desktop-wallet/blob/token-list/src/components/HashEllipsed.tsx)

Some missing padding:

image

Regarding the "?" stuff, I would highly recommend you check out my proposals in the desktop wallet PR, I think they are nice! Looking forward for your input!

File: src/components/Amount.tsx [hunk 1]
```diff
@@ -85,20 +98,22 @@ const Amount = ({
           `${integralPart}.${fractionalPart}`
         ))}
 
-      <Suffix>{suffix && suffix !== 'ALPH' ? ` ${suffix}` : <AlefSymbol color={color} />}</Suffix>
+      <Suffix>{suffix && suffix !== 'ALPH' ? ` ${suffix}` : ' ALPH'}</Suffix>
```
[the remark this answers is not in the document]
Comment (mvaivre) [reply src/components/Amount.tsx:101]: I'm still hesitant, but I figured that it gave ALPHs less importance visually speaking compared to other tokens. Alignments were uglier as well.

File: src/components/Amount.tsx [hunk 2]
```diff
@@ -38,42 +37,56 @@ const Amount = ({
...
     }
-    const amountParts = amount.split('.')
-    integralPart = amountParts[0]
-    fractionalPart = amountParts[1]
+  } else {
+    return (
+      <span className={className} tabIndex={tabIndex ?? -1}>
+        {prefix}
+        {value?.toString()}
+        <Suffix> ???</Suffix>
```
[the remark this answers is not in the document]
Comment (mvaivre) [reply src/components/Amount.tsx]: Done!

**Comment by mvaivre** [comment]
Everything has been addressed. Ready for the final review :)

**Review by nop33 — COMMENTED** [review]
Nice job! 👏 

Is this effect intentional? My guess is that it's not.


https://user-images.githubusercontent.com/1579899/226275154-15f955d3-50f1-45a7-82b0-e3b17b3d7950.mp4


https://user-images.githubusercontent.com/1579899/226275344-a70802dd-6bb7-4aba-b851-4e86e2e02878.mp4

address: `1EckT33Znpgimrwo4XbCSyybuUmGZSAc8cE27BUe6uQTC`

**Comment by mvaivre** [comment]
The issue is fixed in the follow-up PR! Thanks for your review, merging.

**Pull request merged**